portfolio.html

```diff
@@ -8,6 +8,284 @@
     <link href="portfolio.css" rel="stylesheet">
 </head>
 <body>
+
+    <!-- Background images first section -->
+    <div class="background-images-desktop">
+        <img src="./images/desktop_version/frame_headline/Frame.png" class="desktop-frame" alt="Frame group of icon">
+        <img src="./images/desktop_version/frame_headline/Group-38.png" class="desktop-group-38" alt="Group 38 image">
+        <img src="./images/desktop_version/frame_headline/Group2.png" class="desktop-group-2" alt="Group 2 image">
+        <img src="./images/desktop_version/frame_headline/Frame-big.png" class="desktop-frame-big" alt="Frame big icon">
+        <img src="./images/desktop_version/frame_headline/Group-37.png" class="desktop-group-37" alt="Group 37 image">
+        <img src="./images/desktop_version/frame_headline/Frame-middle.png" class="desktop-frame-middle" alt="Frame middle icon">
+        <img src="./images/desktop_version/frame_headline/Frame-right.png" class="desktop-frame-right" alt="Frame right icon">
+        <img src="./images/desktop_version/frame_headline/Group-35.png" class="desktop-group-35" alt="Group 35 image">
```

Comment:
- [ ] Images should be introduced as background images in the CSS file of the particular container that you use, and not through HTML. Please be reminded that you should use HTML only for images that have an important meaning in your app(similar to project images), but not for background images